src/app/functions/server/onerep.ts

```diff
@@ -154,6 +154,23 @@ export async function activateProfile(profileId: number): Promise<void> {
   }
 }
 
+export async function deactivateProfile(profileId: number): Promise<void> {
```

Comment:
Same as above, https://github.com/mozilla/blurts-server/pull/3307 implements this if you want to use it (it's basically identical to yours)